api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -82,28 +80,11 @@ public static class BillingConfig {
     // The environment-driven prefix to apply to Terra billing projects we create. Example:
     // "aou-rw-stable-" causes us to create projects named like "aou-rw-stable-8aec175b".
     public String projectNamePrefix;
-    // The free tier GCP billing account ID to associate with Terra / GCP projects.
+    // The initial credits GCP billing account ID to associate with Terra / GCP projects.
     public String accountId;
 
-    // The legacy free tier billing account id that is migrating away. This value helps to make
-    // migration process smooth.
-    // Null if not set in Config.
-    @Nullable public String legacyAccountId;
-
-    public String freeTierBillingAccountName() {
-      return "billingAccounts/" + accountId;
-    }
-
-    public Optional<String> legacyFreeTierBillingAccountName() {
-      return Optional.ofNullable(legacyAccountId).map(a -> "billingAccounts/" + a);
-    }
-
-    /// All valid free tier billing accounts, including accountId and legacyAccountId(if present).
-    public Set<String> freeTierBillingAccountNames() {
-      Set<String> billingAccountNames = new HashSet<>();
-      billingAccountNames.add(freeTierBillingAccountName());
-      legacyFreeTierBillingAccountName().ifPresent(billingAccountNames::add);
-      return billingAccountNames;
+    public String initialCreditsBillingAccountName() {
+      return fullBillingAccountName(accountId);
```

Comment:
This is the replacement for `freeTierBillingAccountName()` and lives in the new BillingUtils